concourse/tasks/upload-test-artifacts.yml

```diff
@@ -0,0 +1,24 @@
+---
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: governmentpaas/awscli
+inputs:
+  - name: paas-cf
+  - name: artifacts
+run:
+  path: sh
+  args:
+    - -e
+    - -c
+    - |
+      echo "Archiving test artifacts..."
+      RAND=$(hexdump -n 2 -e '/2 "%u"' /dev/urandom)
+      ARTIFACT_NAME="test-artifact-$(date '+%Y-%m-%d-%H-%M-%S')-$RAND.tgz"
+      tar -C artifacts -czvf "${ARTIFACT_NAME}" ./
```

Comment:
It looks like tar doesn't like "." :

```
$ tar xvzf test-artifact-2016-11-03-12-19-56-17410.tgz
x ./: Can't update time for .
x ./acceptance_tests/
x ./acceptance_tests/CATS-TRACE-CATS-1.txt.log
x ./acceptance_tests/CATS-TRACE-CATS-2.txt.log
x ./acceptance_tests.log
x ./acceptance_tests.err.log
tar: Error exit delayed from previous errors.
```

I would suggest something more explicit such as:

```
(cd artifacts && tar -czvf "${ARTIFACT_NAME}" *)
```